buildSrc/src/main/groovy/org/elasticsearch/gradle/test/StandaloneTestPlugin.groovy

```diff
@@ -46,5 +46,10 @@ public class StandaloneTestPlugin implements Plugin<Project> {
         test.testClassesDir project.sourceSets.test.output.classesDir
         test.mustRunAfter(project.precommit)
         project.check.dependsOn(test)
+
+        project.namingConventionsCheck {
+            // Stand alone tests don't follow the rules!
+            skipIntegTestInDisguise = true
```

Comment:
They also are ESIntegTestCase in some cases. Would you prefer I push this change to each violating stand alone test?